src/App.css

```diff
@@ -61,7 +57,9 @@ span.empty {
   box-sizing: border-box;
   color: #666;
   font-family: Verdana, Geneva, Tahoma, sans-serif;
-  background: #fefefe; }
+  background: #fefefe;
```

Comment:
Ideally, colors should be moved into variables and then referenced.

Reply:
Even if the colors are only referenced in only one place?

Reply:
In theory yes because it makes it more extendable and you can give it a name to make it more clear what the color palette is or what you are using it for. In this case, it's fine. I don't do it for one-offs.